seive.js

```diff
@@ -1,31 +1,25 @@
 'use strict';
 
+let primes = [];
```

Comment:
Better not to use a global variable. I suggest making an additional function `findPrimes` taking the result of `seiveOriginal` and returning the primes array.